querybook/webapp/const/options.tsx

```diff
@@ -0,0 +1,5 @@
+export interface OptionsType {
```

Comment:
can you make the one from EnvironmentsSelect.tsx also use this? and also the one in Select.tsx